data/sqlutil/sql.go

```diff
@@ -70,3 +88,85 @@ func FrameFromRows(rows *sql.Rows, rowLimit int64, converters ...Converter) (*da
 
 	return frame, nil
 }
+
+// RunQuery sends the query to the connection and converts the rows to a dataframe.
+func RunQuery(ctx context.Context, db Connection, converters []Converter, fillMode *data.FillMissing, query *Query, args ...interface{}) (data.Frames, error) {
+	// Query the rows from the database
+	rows, err := db.QueryContext(ctx, query.RawSQL, args...)
```

Comment:
This function is not usable for sql data sources that don't use the gosql interface (AKA reshift and athena when we're done with our refactoring). sqlds has, and will always have, a strong dependency to gosql. Wouldn't it make sense to keep functions and types that are tied to gosql in sqlds?

Reply:
Are we planning to stop using the aws provided libraries? This function basically just calls `QueryContext` and does some standard error handling and frame conversion, so I'd imagine we'd still need it. As part of the long running queries work this was separated from the retrying code, which is the part we don't reuse for long running queries. Its copied from `QueryDB` which is used in the aws-sdk for the long running queries code.

Reply:
>Are we planning to stop using the aws provided libraries? 

Not sure what you mean. I'd like us to still use the aws-sdk-go (and grafana-aws-sdk), but I don't want the future async query data handler to have a dependency to gosql as it is synchronous, and the async data handler needs to be asynchronous. I think in this example, it's just the `Connection` interface that concerns me as it returns `sql.Rows`.

Reply:
Ah, I think I get what you mean. I was ok with it returning sql.Rows, since it would be in the sqlutil library, where I thought sql would be assumed, and even if we use it in `aws-sdk` it doesn't mean that we have to use it in the genaralized async query data handler. However, we don't need to move it over (`aws-sdk` will still be dependent on sqlds.Options) so I'm fine closing this pr instead of merging.